### PR TITLE
feat(https): support serving application on TSL

### PR DESCRIPTION
**Pull request by bankyadam**

### CODE REVIEW

**Comment by drawain** [comment]
A sentence is not finished in the readme: "The port for https will be the"

File: app/index.js [hunk 1]
```diff
@@ -88,8 +91,24 @@ App.prototype = {
...
   listen: function(port, env) {
-    this.koaApp.listen(port);
-    console.log('Application started:', { port: port, env: env });
+    var httpPort = parseInt(port);
+    var httpsPort = httpPort + 10000;
+
+    http.createServer(this.koaApp.callback()).listen(httpPort);
+    console.log('Application started:', { port: httpPort, env: env });
+
+    if (process.env.SERVE_HTTPS === 'true') {
```
Comment (drawain) [inline app/index.js:97]: Could you refactor this code to a separate method like "_listenHTTPS" or something like that?

**Comment by bankyadam** [comment]
Made the requested documentation and refactoring requests.

**Comment by drawain** [comment]
LGTM :) Approved

**Pull request merged**